From d7adf3c3d9ccea3b365c5a65d4b2878f65a36de1 Mon Sep 17 00:00:00 2001
From: David Daney <david.daney@cavium.com>
Date: Tue, 15 Jul 2014 18:05:50 -0700
Subject: [PATCH 798/974] MIPS: OCTEON: Disable some cn78xx error interrupt
 conditions.

CVMX_PKI_GEN_INT[DRP_NOAVAIL, CRCERR, BCKPRS] are all normal
conditions and should not cause messages to be emitted.

Signed-off-by: David Daney <david.daney@cavium.com>
[Original patch taken from Cavium SDK 3.1.2-568]
Signed-off-by: Quanyang Wang <quanyang.wang@windriver.com>
---
 arch/mips/cavium-octeon/octeon-78xx-errors.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/mips/cavium-octeon/octeon-78xx-errors.c b/arch/mips/cavium-octeon/octeon-78xx-errors.c
index d0534a8..942b7cb 100644
--- a/arch/mips/cavium-octeon/octeon-78xx-errors.c
+++ b/arch/mips/cavium-octeon/octeon-78xx-errors.c
@@ -1817,12 +1817,12 @@ struct cvmx_error_78xx error_array_cn78xxp1[] = {
 	{ 0x1180038001630ull,	"Error: CVMX_ZIP_QUE6_ERR_INT[%llx][%u]",	0x380c6,	0,	CVMX_ERROR_GROUP_INTERNAL,	1}	/* ECC_DBE */,
 	{ 0x1180038001638ull,	"Error: CVMX_ZIP_QUE7_ERR_INT[%llx][%u]",	0x380c7,	0,	CVMX_ERROR_GROUP_INTERNAL,	1}	/* ECC_DBE */,
 //	{ 0x1180044000020ull,	"Error: CVMX_PKI_GEN_INT[%llx][%u]",	0x44000,	0,	CVMX_ERROR_GROUP_INTERNAL,	0}	/* PKTDRP */,
-	{ 0x1180044000020ull,	"Error: CVMX_PKI_GEN_INT[%llx][%u]",	0x44001,	0,	CVMX_ERROR_GROUP_INTERNAL,	1}	/* CRCERR */,
-	{ 0x1180044000020ull,	"Error: CVMX_PKI_GEN_INT[%llx][%u]",	0x44002,	0,	CVMX_ERROR_GROUP_INTERNAL,	2}	/* BCKPRS */,
+//	{ 0x1180044000020ull,	"Error: CVMX_PKI_GEN_INT[%llx][%u]",	0x44001,	0,	CVMX_ERROR_GROUP_INTERNAL,	1}	/* CRCERR */,
+//	{ 0x1180044000020ull,	"Error: CVMX_PKI_GEN_INT[%llx][%u]",	0x44002,	0,	CVMX_ERROR_GROUP_INTERNAL,	2}	/* BCKPRS */,
 	{ 0x1180044000020ull,	"Error: CVMX_PKI_GEN_INT[%llx][%u]",	0x44003,	0,	CVMX_ERROR_GROUP_INTERNAL,	3}	/* SOP */,
 	{ 0x1180044000020ull,	"Error: CVMX_PKI_GEN_INT[%llx][%u]",	0x44004,	0,	CVMX_ERROR_GROUP_INTERNAL,	4}	/* EOP */,
 	{ 0x1180044000020ull,	"Error: CVMX_PKI_GEN_INT[%llx][%u]",	0x44005,	0,	CVMX_ERROR_GROUP_INTERNAL,	5}	/* DAT */,
-	{ 0x1180044000020ull,	"Error: CVMX_PKI_GEN_INT[%llx][%u]",	0x44006,	0,	CVMX_ERROR_GROUP_INTERNAL,	6}	/* DRP_NOAVAIL */,
+//	{ 0x1180044000020ull,	"Error: CVMX_PKI_GEN_INT[%llx][%u]",	0x44006,	0,	CVMX_ERROR_GROUP_INTERNAL,	6}	/* DRP_NOAVAIL */,
 	{ 0x1180044000020ull,	"Error: CVMX_PKI_GEN_INT[%llx][%u]",	0x44007,	0,	CVMX_ERROR_GROUP_INTERNAL,	7}	/* X2P_REQ_OFL */,
 	{ 0x1180044000040ull,	"Error: CVMX_PKI_ECC_INT0[%llx][%u]",	0x44040,	0,	CVMX_ERROR_GROUP_INTERNAL,	0}	/* ASM_SBE */,
 	{ 0x1180044000040ull,	"Error: CVMX_PKI_ECC_INT0[%llx][%u]",	0x44041,	0,	CVMX_ERROR_GROUP_INTERNAL,	1}	/* ASM_DBE */,
-- 
2.6.2

